.idea/inspectionProfiles/Project_Default.xml

```diff
@@ -31,6 +31,18 @@
     </inspection_tool>
     <inspection_tool class="GwtUiFieldErrors" enabled="false" level="ERROR" enabled_by_default="false" />
     <inspection_tool class="GwtUiHandlerErrors" enabled="false" level="ERROR" enabled_by_default="false" />
+    <inspection_tool class="HttpUrlsUsage" enabled="false" level="WEAK WARNING" enabled_by_default="false">
```

Comment:
This is new in IJ 2021.1 EAP, let me know if it's painful to you when using 2020.3 or earlier, in case we can revert.

Reply:
No problems so far